app/settings/account/-components/default-region/component.ts

```diff
@@ -37,40 +56,39 @@ export default class DefaultRegionPane extends Component {
         if (!user) {
             return;
         }
-
+        this.set('user', user);
+        this.changeset = buildChangeset(user, regionValidation, { skipValidate: true });
         yield user.belongsTo('defaultRegion').reload();
     });
 
+    @task({ withTestWaiter: true })
+    updateRegion = task(function *(this: DefaultRegionPane) {
+        this.changeset.validate();
+        if (this.changeset.get('isValid') && this.user) {
```

Comment:
Can we avoid using `.get` whenever possible?
```suggestion
        if (this.changeset.isValid && this.user) {
```